media-capabilities/index.js

```diff
@@ -17,31 +17,28 @@ import { useState, useEffect } from 'react';
 
 const supported = typeof window !== 'undefined' && 'mediaCapabilities' in navigator;
 
-const useMediaCapabilities = (mediaConfig, initialMediaCapabilities = {}) => {
-  initialMediaCapabilities = {
+const useMediaCapabilitiesDecodingInfo = (mediaDecodingConfig, initialMediaCapabilitiesInfo = {}) => {
```

Comment:
@wingleung 

According to [Media Capabilities API](https://developer.mozilla.org/en-US/docs/Web/API/Media_Capabilities_API), I think we can get `MediaCapabilitiesInfo` at real time.
I wonder if the current implementation can handle real time feedback or do we need to implement some subscription as we did in [dropped battery hook](https://github.com/GoogleChromeLabs/react-adaptive-hooks/commit/e9409b1d85bc2ef81878c46c2ddc3c7c82e281a1)?

Reply:
@wingleung 

Do you have some idea for the above comment?

Reply:
I wonder how the real time feedback would work. For a video component we could switch sources of video streams before it's loaded but for upgrading during playback we would need listeners to hook into. I don't see any listeners in the [specs](https://w3c.github.io/media-capabilities/) that we can hook into.

I do see a reference to [Media Playback Quality](https://w3c.github.io/media-capabilities/#biblio-media-playback-quality)

> Real time feedback about the playback so an adaptative streaming can alter the quality of the content based on actual user perceived quality. Such information will allow websites to react to a pick of CPU/GPU usage in real time. It is expected that this will be tackled as part of the [[media-playback-quality]](https://w3c.github.io/media-capabilities/#biblio-media-playback-quality) specification.